aspnetcore/migration/70-80.md

```diff
@@ -73,64 +73,541 @@ In the project file, update each [Microsoft.AspNetCore.*](https://www.nuget.org/
+> [!IMPORTANT]
+> This section focuses on the minimal changes required to update a .NET 7 Blazor Server app to .NET 8. The guidance in this section doesn't cover adoption of the Blazor Web App project template folder and file layout structure, nor does it adhere to new layout, error UI, navigation menu, stylesheet, and other changes introduced for Blazor Web Apps. To adopt all of the introduced changes for .NET 8, convert the Blazor Server app into a Blazor Web App by following the guidance in the [Convert a Blazor Server app into a Blazor Web App](#convert-a-blazor-server-app-into-a-blazor-web-app) section. 
 
-In .NET 6 and earlier:
+1. Follow the guidance in the first three sections of this article:
 
-```razor
-<NestedGrandchild @bind-GrandchildMessage="BoundValue" />
+   * [Update .NET Core SDK version in `global.json`](#update-net-core-sdk-version-in-globaljson)
+   * [Update the target framework](#update-the-target-framework)
+   * [Update package references](#update-package-references)
 
-@code {
-    ...
+1. Move the contents of the `App` component (`App.razor`) to a new `Routes` component file (`Routes.razor`) added to the project's root folder. Leave the empty `App.razor` file in the app in the project's root folder.
 
-    private string BoundValue
-    {
-        get => ChildMessage ?? string.Empty;
-        set => ChildMessageChanged.InvokeAsync(value);
-    }
-}
-```
+1. Add an entry to the `_Imports.razor` file to make shorthand render modes available to the app:
 
-In .NET 7, you can use the new `@bind:get` and `@bind:set` modifiers to support two-way data binding and simplify the binding syntax:
+   ```razor
+   @using static Microsoft.AspNetCore.Components.Web.RenderMode
+   ```
 
-```razor
-<NestedGrandchild @bind-GrandchildMessage:get="ChildMessage" 
-    @bind-GrandchildMessage:set="ChildMessageChanged" />
-```
+1. Move the content in the `_Host` page (`Pages/_Host.cshtml`) to the empty `App.razor` file. Proceed to make the following changes to the `App` component.
 
-> [!IMPORTANT]
-> The `@bind:get` and `@bind:set` features are receiving further updates at this time. To take advantage of the latest updates, confirm that you've installed the [latest SDK](https://dotnet.microsoft.com/download/dotnet/7.0).
->
-> Using an event callback parameter (`[Parameter] public EventCallback<string> ValueChanged { get; set; }`) isn't supported. Instead, pass an <xref:System.Action>-returning or <xref:System.Threading.Tasks.Task>-returning method to `@bind:set`.
->
-> For more information, see the following resources:
->
-> * [Blazor `@bind:after` not working on .NET 7 RTM release (dotnet/aspnetcore #44957)](https://github.com/dotnet/aspnetcore/issues/44957)
-> * [`BindGetSetAfter701` sample app (javiercn/BindGetSetAfter701 GitHub repository)](https://github.com/javiercn/BindGetSetAfter701)
+   > [!NOTE]
+   > In the following example, the project's namespace is `Net8BlazorServer`. Adjust the namespace to match your project.
```

Comment:
Replace `Net8BlazorServer` with `Net7BlazorServer` to clarify that the app is initially a .NET 7 app.

Reply:
Perhaps, we should remove the number. It seems confusing either way.

Reply:
Sure, how about `BlazorServerApp`?